libs/application/templates/README.md

```diff
@@ -1,5 +1,33 @@
 # Templates
 
+## Pruning notifications
+
+If desired it is possible to have a custom message sent to an application's applicant when it is pruned.
+This is done by adding a `pruneMessage` object of type `PruningNotification` or a function returning a `PruningNotification` to the application's lifecycle configuration.
+The function will be passed an argument containing a `PruningApplication` object which contains the application's data. The `PruningNotification` object has `externalBody`, `internalBody` and `notificationTemplateId`. The former two can be used to fill in the values for the `externalBody` and `internalBody` template variables in the notification that will be sent to the user. The `notificationTemplateId` is the id of the notification template that will be used to send the notification. It is up to the user whether to use the body variables or not.
```

Comment:
```suggestion
The function will be passed as an argument containing a `PruningApplication` object which contains the application's data. The `PruningNotification` object has `externalBody`, `internalBody` and `notificationTemplateId`. The former two can be used to fill in the values for the `externalBody` and `internalBody` template variables in the notification that will be sent to the user. The `notificationTemplateId` is the id of the notification template that will be used to send the notification. It is up to the user whether to use the body variables or not.
```

Reply:
The intention here was to say that when the function gets executed it gets passed an argument that is a PruningApplication object. 
I have updated the relevant text to try to make that clearer, thanks for pointing this out.